gluoncv/nn/splat.py

```diff
@@ -0,0 +1,64 @@
+import mxnet as mx
+from mxnet.gluon import nn
+from mxnet.gluon.nn import Conv2D, Block, HybridBlock, Dense, BatchNorm, Activation
+
+__all__ = ['SplitAttentionConv']
+
+USE_BN = True
+
+class SplitAttentionConv(HybridBlock):
+    def __init__(self, channels, kernel_size, strides=(1, 1), padding=(0, 0),
+                 dilation=(1, 1), groups=1, radix=2, *args, in_channels=None, r=2,
+                 norm_layer=BatchNorm, norm_kwargs=None, drop_ratio=0, **kwargs):
+        super().__init__()
+        norm_kwargs = norm_kwargs if norm_kwargs is not None else {}
+        inter_channels = max(in_channels*radix//2//r, 32)
+        self.radix = radix
+        self.cardinality = groups
+        self.conv = Conv2D(channels*radix, kernel_size, strides, padding, dilation,
+                           groups=groups*radix, *args, in_channels=in_channels, **kwargs)
+        if USE_BN:
+            self.bn = norm_layer(in_channels=channels*radix, **norm_kwargs)
+        self.relu = Activation('relu')
+        self.fc1 = Conv2D(inter_channels, 1, in_channels=channels, groups=self.cardinality)
+        if USE_BN:
```

Comment:
maybe `if norm_layer is not None` is sufficient and more flexible?

Reply:
Agree. Just changed it.